TTHAnalysis/python/tools/nanoAOD/ttH_modules.py

```diff
@@ -134,7 +134,7 @@ def clean_and_FO_selection_TTH(lep,year):
 countTaus = lambda : ObjTagger('Tight','TauSel_Recl', [lambda t : t.idDeepTau2017v2p1VSjet&4])
 
 from PhysicsTools.NanoAODTools.postprocessing.modules.jme.jetmetUncertainties import jetmetUncertainties2016All,jetmetUncertainties2017All,jetmetUncertainties2018All
```

Comment:
I seem to remember that the changes are because of something that is supposed to be temporary, right? I would leave this uncommented for the moment; just put the updated file in your web area and mention to Derek that for the moment temporarily he has to pick that file replacing the original from the branch